great_expectations/rule_based_profiler/data_assistant_result/data_assistant_result.py

```diff
@@ -2762,10 +2715,10 @@ def _get_interactive_expect_column_values_to_be_between_line_chart(
             points.selection = alt.Undefined
             points.transform = alt.Undefined
 
-            point_color_condition: alt.condition = alt.condition(
+            point_color_condition = alt.condition(
                 predicate=predicates[idx],
-                if_false=alt.value(Colors.GREEN.value),
-                if_true=alt.value(Colors.PINK.value),
+                if_false=alt.value(Colors.PINK.value),
```

Comment:
good catch!

Reply:
This needed to be updated after changing `predicates` to use positive logic.